interp/constrextern.ml

```diff
@@ -479,7 +479,8 @@ and extern_notation_pattern (tmp_scope,scopes as allscopes) vars t = function
       if is_inactive_rule keyrule then raise No_match;
       let loc = t.loc in
       match DAst.get t with
-	| PatCstr (cstr,_,na) ->
+        | PatCstr (cstr,args,na) ->
+          let t = if na = Anonymous then t else DAst.make ?loc (PatCstr (cstr,args,Anonymous)) in
```

Comment:
Why not use `is_anonymous`?

Reply:
It could be `is_anonymous` (I did not remember it existed). I actually feel naive, are there particular implications at using one or the other here?